test/docfx.Test/lib/GitHubUserCacheTest.cs

```diff
@@ -126,6 +126,28 @@ private static IEnumerable<object[]> PrepareDataCore()
                 1
             };
             yield return new object[]
+            {
+                "Get user by commit does not return user not found error",
+                (Func<GitHubUserCache, Task>) ( async (cache) =>
+                {
+                    Assert.Null((await cache.GetByCommit("me@contoso.com", "owner", "name", "3")).error);
```

Comment:
seems a bit odd to assert here, possible to return it and assert in `TestCache`?